svc-mesh/service-mesh.md

```diff
@@ -0,0 +1,229 @@
+# Kata Containers and service mesh for Kubernetes
+
+A service mesh is a powerful way to monitor and control the traffic
+between micro-services running in your Kubernetes cluster. They are
+powerful tools that you might want to use in combination with the
+security brought by Kata Containers.
+
+## How do they work ?
+
+They both rely on the same model, where they run controller applications
+in the control plane, and inject a proxy as a sidecar inside the service
+pod. The proxy registers to the control plane as a first step, and it
```

Comment:
The proxy registers in the control plane...

Reply:
done